cmd/otel-allocator/internal/config/config.go

```diff
@@ -38,6 +40,17 @@ const (
 	DefaultCollectorNotReadyGracePeriod                = 0 * time.Second
 )
 
+// By default, scrape protocols include PrometheusText1_0_0, which only Prometheus >=3.0 supports.
+// Manually exclude this protocol until several versions of the Otel Collector support it.
```

Comment:
should we specify a version here and make an issue for updating this?

Reply:
https://github.com/open-telemetry/opentelemetry-operator/issues/3872